### PR TITLE
Window center, minimum and maximum

### PR DESCRIPTION
**Pull request by Zack-83**
Solves #131 
Solves #88

### CODE REVIEW

File: src/templates/vibso_classes.tsv [hunk 1]
```diff
@@ -15,8 +15,8 @@ obo:VIBSO_0000011	diffraction grating		https://github.com/NFDI4Chem/VibrationalS
...
+obo:VIBSO_0000015	spectral window range maximum	acquisition band maximum	https://github.com/NFDI4Chem/VibrationalSpectroscopyOntology/issues/31	A setting datum which determines the maximum wavenumber included in the spectrum.		owl:Class	data item|('part of' some 'Raman spectrum')	ORCID:0000-0002-1595-3213|ORCID:0000-0001-7694-5519	210 cm^-1		It needs to be determined, if this is also part of another kind of spectrum, such as a vibrational spectrum in general. We also need to decide, if we need to also model the terms window size and window center, from which this term is derived.
```
Comment (StroemPhi) [inline src/templates/vibso_classes.tsv:18]: So far we always used the as wording for the Genus "A setting datum that specifies ...". And how about we define it as not being dependent on being quantized as a wavenumber? My proposal to rewrite to: 
A setting datum that specifies the upper boundary of the spectral window range utilized during the acquisition of spectroscopic data.

File: src/templates/vibso_classes.tsv [hunk 2]
```diff
@@ -15,8 +15,8 @@ obo:VIBSO_0000011	diffraction grating		https://github.com/NFDI4Chem/VibrationalS
...
+obo:VIBSO_0000015	spectral window range maximum	acquisition band maximum	https://github.com/NFDI4Chem/VibrationalSpectroscopyOntology/issues/31	A setting datum which determines the maximum wavenumber included in the spectrum.		owl:Class	data item|('part of' some 'Raman spectrum')	ORCID:0000-0002-1595-3213|ORCID:0000-0001-7694-5519	210 cm^-1		It needs to be determined, if this is also part of another kind of spectrum, such as a vibrational spectrum in general. We also need to decide, if we need to also model the terms window size and window center, from which this term is derived.			
+obo:VIBSO_0000016	spectral window range minimum	acquisition band minimum	https://github.com/NFDI4Chem/VibrationalSpectroscopyOntology/issues/31	A setting datum which determines the minimum wavenumber included in the spectrum.		owl:Class	data item|('part of' some 'Raman spectrum')	ORCID:0000-0002-1595-3213|ORCID:0000-0001-7694-5519	1960 cm^-1		It needs to be determined, if this is also part of another kind of spectrum, such as a vibrational spectrum in general. We also need to decide, if we need to also model the terms window size and window center, from which this term is derived.
```
Comment (StroemPhi) [inline src/templates/vibso_classes.tsv:19]: My proposal to rewrite to:
A setting datum that specifies the lower boundary of the spectral window range utilized during the acquisition of spectroscopic data.

File: src/templates/vibso_classes.tsv [hunk 3]
```diff
@@ -46,4 +46,4 @@ obo:VIBSO_0000042	spectral resolution interval as energy		https://github.com/NFD
...
-											
\ No newline at end of file
+obo:......	spectral window center		https://github.com/NFDI4Chem/VibrationalSpectroscopyOntology/issues/88	A setting datum which determines the middle wavelength/wavenumber of a spectrum and corresponds to the centering of the spectrometer. It is the average of the spectral window range maximum and minimum.		owl:Class		ORCID:0000-0002-1595-3213|ORCID:0000-0002-2239-3955
```
Comment (StroemPhi) [inline src/templates/vibso_classes.tsv:49]: My proposal to rewrite to:
A setting datum that specifies the average of the maximum and minimum of the spectral window range utilized during the acquisition of spectroscopic data, and that corresponds to the optimal alignment or centering of the spectrometer to ensure that the most relevant spectral features are captured effectively within the defined acquisition range.

We should also add an example similar to the minimum and maximum.

**Review by StroemPhi — CHANGES_REQUESTED** [review]

File: src/templates/vibso_classes.tsv (hunk 3 above)
Comment (Zack-83) [reply src/templates/vibso_classes.tsv:49]:
```suggestion
obo:......	spectral window center		https://github.com/NFDI4Chem/VibrationalSpectroscopyOntology/issues/88	A setting datum that specifies the middle wavelength/wavenumber of a spectrum, equal to the average of the maximum and minimum of the spectral window range utilized during the acquisition of spectroscopic data. It corresponds to the optimal alignment or centering of the spectrometer to ensure that the most relevant spectral features are captured effectively within the defined acquisition range.		owl:Class		ORCID:0000-0002-1595-3213|ORCID:0000-0002-2239-3955				
```

File: src/templates/vibso_classes.tsv (hunk 2 above)
Comment (Zack-83) [reply src/templates/vibso_classes.tsv:19]:
```suggestion
obo:VIBSO_0000016	spectral window range minimum	acquisition band minimum	https://github.com/NFDI4Chem/VibrationalSpectroscopyOntology/issues/31	A setting datum that specifies the lower boundary of the spectral window range utilized during the acquisition of spectroscopic data.		owl:Class	data item|('part of' some 'Raman spectrum')	ORCID:0000-0002-1595-3213|ORCID:0000-0001-7694-5519	1960 cm^-1		It needs to be determined, if this is also part of another kind of spectrum, such as a vibrational spectrum in general. We also need to decide, if we need to also model the terms window size and window center, from which this term is derived.			
```

File: src/templates/vibso_classes.tsv [hunk 4]
```diff
@@ -15,8 +15,8 @@ obo:VIBSO_0000011	diffraction grating		https://github.com/NFDI4Chem/VibrationalS
...
+obo:VIBSO_0000015	spectral window range maximum	acquisition band maximum	https://github.com/NFDI4Chem/VibrationalSpectroscopyOntology/issues/31	A setting datum that specifies the upper boundary of the spectral window range utilized during the acquisition of spectroscopic data.		owl:Class	data item|('part of' some 'Raman spectrum')	ORCID:0000-0002-1595-3213|ORCID:0000-0001-7694-5519	210 cm^-1		It needs to be determined, if this is also part of another kind of spectrum, such as a vibrational spectrum in general. We also need to decide, if we need to also model the terms window size and window center, from which this term is derived.
```
Comment (StroemPhi) [inline src/templates/vibso_classes.tsv:18]:
```suggestion
obo:VIBSO_0000015	spectral window range maximum	acquisition band maximum	https://github.com/NFDI4Chem/VibrationalSpectroscopyOntology/issues/31	A setting datum that specifies the upper boundary of the spectral window range utilized during the acquisition of spectroscopic data.		owl:Class	setting datum	ORCID:0000-0002-1595-3213|ORCID:0000-0001-7694-5519	210 cm^-1		It needs to be determined, if this is also part of another kind of spectrum, such as a vibrational spectrum in general. We also need to decide, if we need to also model the terms window size and window center, from which this term is derived.			
```

File: src/templates/vibso_classes.tsv [hunk 5]
```diff
@@ -15,8 +15,8 @@ obo:VIBSO_0000011	diffraction grating		https://github.com/NFDI4Chem/VibrationalS
...
+obo:VIBSO_0000016	spectral window range minimum	acquisition band minimum	https://github.com/NFDI4Chem/VibrationalSpectroscopyOntology/issues/31	A setting datum that specifies the lower boundary of the spectral window range utilized during the acquisition of spectroscopic data.		owl:Class	data item|('part of' some 'Raman spectrum')	ORCID:0000-0002-1595-3213|ORCID:0000-0001-7694-5519	1960 cm^-1		It needs to be determined, if this is also part of another kind of spectrum, such as a vibrational spectrum in general. We also need to decide, if we need to also model the terms window size and window center, from which this term is derived.
```
Comment (StroemPhi) [inline src/templates/vibso_classes.tsv:19]:
```suggestion
obo:VIBSO_0000016	spectral window range minimum	acquisition band minimum	https://github.com/NFDI4Chem/VibrationalSpectroscopyOntology/issues/31	A setting datum that specifies the lower boundary of the spectral window range utilized during the acquisition of spectroscopic data.		owl:Class	setting datum	ORCID:0000-0002-1595-3213|ORCID:0000-0001-7694-5519	1960 cm^-1		It needs to be determined, if this is also part of another kind of spectrum, such as a vibrational spectrum in general. We also need to decide, if we need to also model the terms window size and window center, from which this term is derived.			
```

File: src/templates/vibso_classes.tsv [hunk 6]
```diff
@@ -46,4 +46,4 @@ obo:VIBSO_0000042	spectral resolution interval as energy		https://github.com/NFD
...
-											
\ No newline at end of file
+obo:......	spectral window center		https://github.com/NFDI4Chem/VibrationalSpectroscopyOntology/issues/88	A setting datum that specifies the middle wavelength/wavenumber of a spectrum, equal to the average of the maximum and minimum of the spectral window range utilized during the acquisition of spectroscopic data. It corresponds to the optimal alignment or centering of the spectrometer to ensure that the most relevant spectral features are captured effectively within the defined acquisition range.		owl:Class		ORCID:0000-0002-1595-3213|ORCID:0000-0002-2239-3955
```
Comment (StroemPhi) [inline src/templates/vibso_classes.tsv:49]:
```suggestion
obo:......	spectral window center		https://github.com/NFDI4Chem/VibrationalSpectroscopyOntology/issues/88	A setting datum that specifies the middle wavelength/wavenumber of a spectrum, equal to the average of the maximum and minimum of the spectral window range utilized during the acquisition of spectroscopic data. It corresponds to the optimal alignment or centering of the spectrometer to ensure that the most relevant spectral features are captured effectively within the defined acquisition range.		owl:Class	setting datum	ORCID:0000-0002-1595-3213|ORCID:0000-0002-2239-3955	560 cm^-1			
```

**Review by StroemPhi — CHANGES_REQUESTED** [review]
Just saw that we need to update the parent as well

**Review by StroemPhi — APPROVED** [review]

**Pull request merged**